utils/db.ts

```diff
@@ -409,6 +412,16 @@ export async function deleteUserBySession(sessionId: string) {
 
 export async function getUsersByIds(ids: string[]) {
   const keys = ids.map((id) => ["users", id]);
-  const res = await kv.getMany<User[]>(keys);
-  return res.map((entry) => entry.value!);
+  // NOTE: limit of 10 for getMany or `TypeError: too many ranges (max 10)`
+  const users: User[] = [];
+  for (const batch of batchify(keys, 10)) {
+    users.push(...(await kv.getMany<User[]>(batch)).map((entry) => entry.value!))
+  }
+  return users
+}
+
+export function* batchify<T>(arr: T[], n = 5): Generator<T[], void> {
+  for (let i = 0; i < arr.length; i += n) {
+    yield arr.slice(i, i + n);
+  }
```

Comment:
Can do. Originally `batchify` lived in the `/tools/submissions.ts` but I ran into issues when trying to call `kv.getMany()` with more than 10 keys (e.g. 30 when I expanded the range of values the page `limit` parameter would accept). Since we have made limit a fixed size of 10 again I will revert this.